### ISSUE TITLE
[VS Mac] Target framework changed unexpectedly after editing storyboard with XCode

### ISSUE THREAD

**Issue by snechaev** [xamarin/xamarin-macios#10010]
Note: this is not Xamarin issue, but VS Mac issue. But on the [Xamarin forums](https://forums.xamarin.com/discussion/184988/target-framework-changes-when-storyboard-is-changed) I was advised to report such an issue here.

### Steps to Reproduce
1. Run VS Mac and create new Cocoa App (for MacOS) project. 
2. Open project settings and change Target framework to a "Xamarin.Mac Full" option (.NET Framework 4.8). Save changes.
3. Right-click the Main.sroryboard and select Open With -> XCode Interface Builder. 
4. When XCode will run, switch focus on the XCode window then back to the VS Mac window
5. Wait until VS Mac finish the XCode sync.
6. Check the Target framework in the project options

### Expected Behavior
The target framework is Xamarin.Mac Full (NET FW4.8)

### Actual Behavior
The target framework is Xamarin.Mac Modern. In the clear new project there is no other side effects, but in the real solution, which contains the .net FW-based projects and libraries, the solution can not be built until you change target framework back to Full.

### Environment

```
=== Visual Studio Professional 2019 for Mac ===

Version 8.7.9 (build 9)
Installation UUID: 81c1605b-4a96-42a5-b9a6-226dd7d9da66
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 6.18.0.23 (d16-6 / 088c73638)

	Package version: 612000093

=== Mono Framework MDK ===

Runtime:
	Mono 6.12.0.93 (2020-02/620cf538206) (64-bit)
	Package version: 612000093

=== Roslyn (Language Service) ===

3.7.0-6.20427.1+18ede13943b0bfae1b44ef078b2f3923159bcd32

=== NuGet ===

Version: 5.7.0.6702

=== .NET Core SDK ===

SDK: /usr/local/share/dotnet/sdk/3.1.403/Sdks
SDK Versions:
	3.1.403
	3.1.402
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/msbuild/Current/bin/Sdks

=== .NET Core Runtime ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	3.1.9
	3.1.8
	2.1.23
	2.1.22

=== Xamarin.Profiler ===

Version: 1.6.15.68
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Updater ===

Version: 11

=== Xamarin.Android ===

Not Installed

=== Microsoft OpenJDK for Mobile ===

Java SDK: Not Found

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Android SDK Manager ===

Version: 16.7.0.13
Hash: 8380518
Branch: remotes/origin/d16-7~2
Build date: 2020-09-16 05:12:24 UTC

=== Android Device Manager ===

Version: 16.7.0.24
Hash: bb090a3
Branch: remotes/origin/d16-7
Build date: 2020-09-16 05:12:46 UTC

=== Xamarin Designer ===

Version: 16.7.0.495
Hash: 03d50a221
Branch: remotes/origin/d16-7-vsmac
Build date: 2020-08-28 13:12:52 UTC

=== Apple Developer Tools ===

Xcode 12.1 (17222)
Build 12A7403

=== Xamarin.Mac ===

Version: 6.20.2.2 (Visual Studio Professional)
Hash: 817b6f72a
Branch: d16-7
Build date: 2020-07-18 18:44:59-0400

=== Xamarin.iOS ===

Version: 14.2.0.12 (Visual Studio Professional)
Hash: e0ce49680
Branch: xcode12.1
Build date: 2020-10-20 09:52:06-0400

=== Build Information ===

Release ID: 807090009
Git revision: d29d54665a1fc79986687a540a7a4676f97ad3da
Build date: 2020-10-21 15:05:18-04
Build branch: release-8.7
Xamarin extensions: d29d54665a1fc79986687a540a7a4676f97ad3da

=== Operating System ===

Mac OS X 10.15.7
Darwin 19.6.0 Darwin Kernel Version 19.6.0
    Mon Aug 31 22:12:52 PDT 2020
    root:xnu-6153.141.2~1/RELEASE_X86_64 x86_64


```

### Build Logs
No build needed to reproduce the issue

### Example Project (If Possible)

[test.zip](https://github.com/xamarin/xamarin-macios/files/5463703/test.zip)

> VS bug [#1240685](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1240685)

**Comment by chamons** [comment]
Thanks for the report. I'm adding a label which will create an internal bug for the VS team and will pass it along to them.

For IDE features you are also welcome to post issues on [Developer Community](https://developercommunity.visualstudio.com/spaces/41/index.html) as well.

**Comment by kunyli** [comment]
@snechaev Hi! This is a duplicate of another issue that has a fix released in version 8.8 Preview 4. Please update VS Mac to receive the fix. Thanks for reporting the issue!
